docs_zh_CN/demo_cn.md

```diff
@@ -0,0 +1,319 @@
+    --det-score-thr 0.9 \
+    --action-score-thr 0.5 \
+    --label-map demo/label_map_ava.txt \
+    --predict-stepsize 8 \
+    --output-stepsize 4 \
+    --output-fps 6
+```
+
+## Video GradCAM Demo
+
+MMAction2 提供如下脚本以可视化输入视频的 GradCAM。
+
+```shell
+python demo/demo_gradcam.py ${CONFIG_FILE} ${CHECKPOINT_FILE} ${VIDEO_FILE} [--use-frames] \
+    [--device ${DEVICE_TYPE}] [--target-layer-name ${TARGET_LAYER_NAME}] [--fps {FPS}] \
+    [--target-resolution ${TARGET_RESOLUTION}] [--resize-algorithm {RESIZE_ALGORITHM}] [--out-filename {OUT_FILE}]
+```
+
+可选参数：
+
+- `--use-frames`: 如指定，代表使用帧目录作为输入；否则代表使用视频作为输入。
+- `DEVICE_TYPE`: 指定脚本运行设备，支持 cuda 设备（如 `cuda:0`）或cpu（`cpu`）。默认为 `cuda:0`。
+- `TARGET_LAYER_NAME`: 需要生成 GradCAM 可视化的网络层名称。
+- `FPS`: 使用帧目录作为输入时，代表输入的帧率。默认为 30。
+- `TARGET_RESOLUTION`: 输出视频的分辨率，如未指定，使用输入视频的分辨率。
+- `RESIZE_ALGORITHM`: 缩放视频时使用的插值方法，默认为 `bilinear`。
+- `OUT_FILE`: 输出视频的路径，如未指定，则不会生成输出视频。
+
+示例：
+
+以下示例假设用户的当前目录为 `$MMACTION2`，并已经将所需的模型检查点文件下载至目录 `checkpoints/` 下，用户也可以使用所提供的 URL 来直接加载模型检查点，文件将会被默认下载至 `$HOME/.cahe/torch/checkpoints`。
+
+1. 对于 I3D 模型进行 GradCAM 的可视化，使用视频作为输入，并输出一帧率为 10 的 GIF 文件：
+
+    ```shell
+    python demo/demo_gradcam.py configs/recognition/i3d/i3d_r50_video_inference_32x2x1_100e_kinetics400_rgb.py \
+        checkpoints/i3d_r50_video_32x2x1_100e_kinetics400_rgb_20200826-e31c6f52.pth demo/demo.mp4 \
+        --target-layer-name backbone/layer4/1/relu --fps 10 \
+        --out-filename demo/demo_gradcam.gif
+    ```
+
+2. 对于 I3D 模型进行 GradCAM 的可视化，使用视频作为输入，并输出一 GIF 文件，此示例利用 URL 加载模型检查点文件：
+
+    ```shell
+    python demo/demo_gradcam.py configs/recognition/tsm/tsm_r50_video_inference_1x1x8_100e_kinetics400_rgb.py \
+        https://download.openmmlab.com/mmaction/recognition/tsm/tsm_r50_video_1x1x8_100e_kinetics400_rgb/tsm_r50_video_1x1x8_100e_kinetics400_rgb_20200702-a77f4328.pth \
+        demo/demo.mp4 --target-layer-name backbone/layer4/1/relu --out-filename demo/demo_gradcam_tsm.gif
+    ```
+
+## Webcam demo
+
+MMAction2 提供如下脚本来进行使用网络摄像头的实时动作识别。为得到 [0, 1] 间的动作分值，请确保在配置文件中设定 `model['test_cfg'] = dict(average_clips='prob')` 。
+
+```shell
+python demo/webcam_demo.py ${CONFIG_FILE} ${CHECKPOINT_FILE} ${LABEL_FILE} \
+    [--device ${DEVICE_TYPE}] [--camera-id ${CAMERA_ID}] [--threshold ${THRESHOLD}] \
+    [--average-size ${AVERAGE_SIZE}] [--drawing-fps ${DRAWING_FPS}] [--inference-fps ${INFERENCE_FPS}]
+```
+
+可选参数：
+
+- `DEVICE_TYPE`: 指定脚本运行设备，支持 cuda 设备（如 `cuda:0`）或cpu（`cpu`）。默认为 `cuda:0`。
+- `CAMERA_ID`: 摄像头设备的 ID，默认为 0。
+- `THRESHOLD`: 动作识别的分数阈值，只有分数大于阈值的动作类型会被显示，默认为 0。
+- `AVERAGE_SIZE`: 使用最近N个片段的平均结果作为预测，默认为 1。
+- `DRAWING_FPS`: 可视化结果时的最高帧率，默认为 20。
+- `INFERENCE_FPS`: 进行推理时的最高帧率，默认为 4。
+
+**注**： 若用户的硬件配置足够，可增大可视化帧率和推理帧率以带来更好体验。
+
+示例：
+
+以下示例假设用户的当前目录为 `$MMACTION2`，并已经将所需的模型检查点文件下载至目录 `checkpoints/` 下，用户也可以使用所提供的 URL 来直接加载模型检查点，文件将会被默认下载至 `$HOME/.cahe/torch/checkpoints`。
```

Comment:
`$HOME/.cahe/torch/checkpoints`  -> `$HOME/.cache/torch/checkpoints`

Reply:
hey, It seems that the original text is also wrong. Could you revise it in this PR?

https://github.com/open-mmlab/mmaction2/blob/master/demo/README.md